src/converter/one/convert/JfrConverter.java

```diff
@@ -61,10 +61,15 @@ protected EventAggregator collectEvents() throws IOException {
 
         long startTicks = args.from != 0 ? toTicks(args.from) : Long.MIN_VALUE;
         long endTicks = args.to != 0 ? toTicks(args.to) : Long.MAX_VALUE;
+        boolean allowZeroes = !args.nativemem;
+        IEventReader reader = eventFilter != null ? eventFilter : jfr;
```

Comment:
Need to revise this approach - it will not work for multi-chunk JFRs.
An address map should persist between chunks.

Reply:
`IEventReader` instance persists between `collectEvents` calls, as it is created in the `JfrConverter` ctor.

Reply:
Went with a different design around the `EventAggregator` class.